templates/back/signalement/view.html.twig

```diff
@@ -454,13 +457,15 @@
                     <div class="fr-col-12 fr-col-md-5">                    
                         <strong>Analyse :</strong> <span class="{{ signalementQualificationNDE.status|status_to_css }}">{{ signalementQualificationNDE.status.label }}</span>
                     </div>
-                    
-                    <div class="fr-col-12 fr-col-md-2 fr-text--right">
-                        <button class="fr-btn fr-btn--secondary fr-fi-edit-line fr-btn--icon-left" data-fr-opened="false"
-                                aria-controls="fr-modal-edit-nde">
-                            Modifier
-                        </button>
-                    </div>
+                                  
+                    {% if signalement.statut != constant('App\\Entity\\Signalement::STATUS_NEED_VALIDATION') and signalement.statut != constant('App\\Entity\\Signalement::STATUS_REFUSED') and not isClosed and not isClosedForMe and (isAccepted or is_granted('ROLE_ADMIN_TERRITORY') ) %}

```

Comment:
Vu que les deux conditions se ressemblent, tu peux regrouper dans une seule variable (canEditNDE ?) en PHP ?
Sinon il y a le risque de ne corriger un seul endroit si un cas change

Par ailleurs, l'accumulation des statuts, c'est pas juste statut == ACTIVE ?
Là ça laisse la possibilité de l'éditer quand il est ARCHIVED (mais c'est peut être volontaire)

Reply:
ok

Reply:
Je partage elle est complexe la condition. ça serait pas signalement actif et non fermé pour moi
Le dernier bloc de condition je comprend pas trop ` and (isAccepted or is_granted('ROLE_ADMIN_TERRITORY') `

Reply:
Pas tout à fait, parce que si c'est un admin ou admin territoire, il faut pouvoir éditer même si c'est fermé pour soi